DEV/Login/LoginViewContoller.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  LoginViewContoller.swift
+//  DEV
+//
+//  Created by Ceri-anne Jackson on 27/08/2018.
+//  Copyright © 2018 DEV. All rights reserved.
+//
+
+import Foundation
+import WebKit
+
+class LoginViewController: DevWebViewController {
+    
+    override func viewDidLoad() {
+        self.webURL = DevServiceURL.login.fullURL
+        super.viewDidLoad()
+    }
+    
+}
+
+extension LoginViewController {
+    
+    func webView(_ webView: WKWebView, didReceiveServerRedirectForProvisionalNavigation navigation: WKNavigation!) {
+       
+        if let urlString = webView.url?.absoluteString {
+            redirect(to: urlString)
+        }
+        
+    }
+    
+    func redirect(to urlString: String) {
+        
+        if !loggedIn(urlString: urlString) {
```

Comment:
This would be a good candidate for a guard statement since if this condition isn't valid then we shouldn't continue
```swift
guard isLoggedIn(urlString: urlString) else { return }
```